### PR TITLE
Spain bros

### PR DESCRIPTION
**Pull request by ghost**
1. Added 3 provinces from Central America to the correct **continent** (they were in South America for some reason).
2. Reworked Spanish Viceroyalty decisions.
3. New Spain on formation will automatically annex any vassals Spain has from North America (e.g. Yacutan, Rio Grande, Texas).
4. La Plata can be formed without Uruguay and Paraguay, but they will still get integrated if they are a puppet of Spain.
5. Re-ordered decisions based on importance.

### CODE REVIEW

**Comment by ghost** [comment]
6. Remade **Democratize The Viceroyalties** decision, to become available again when a new Viceroyalty has been established.

**Comment by jajang04** [comment]
is it playable?

**Comment by ghost** [comment]
yes, although i noticed that **annex_to** creates ghost units if the nation that gets annexed start building them before the annexation

File: TGC/decisions/SPA.txt [hunk 1]
```diff
@@ -72,1738 +127,1066 @@ political_decisions = {
...
 				}
 			}
-		}
 
-		allow = {
-			any_owned_province = { region = SPA_1455 }
-			invention = mission_to_civilize
-			money = 50000
-		}
+			URU = all_core = { add_core = VLP }
```
Comment (joelAVS) [inline TGC/decisions/SPA.txt:676]: Those two lines are incorrect, missing the outer brackets, causing a parse error. The correct code should go:
URU = { all_core = { add_core = VLP } }
PRG = { all_core = { add_core = VLP } }

File: TGC/decisions/SPA.txt [hunk 2]
```diff
@@ -1,5 +1,60 @@
-political_decisions = {
-#from SPA.txt
+﻿political_decisions = {
```
Comment (joelAVS) [inline TGC/decisions/SPA.txt:1]: The whole file is in the incorrect encoding, it needs to be re-saved as win 1252.

**Pull request closed without merging**